src/OgGroupAudienceHelper.php

```diff
@@ -111,4 +118,101 @@ public static function getMatchingField(ContentEntityInterface $entity, $group_t
     return NULL;
   }
 
+  /**
+   * Get list of available widgets.
+   *
+   * @return array
+   *   List of available entity reference widgets.
+   */
+  public static function getAvailableWidgets() {
+    $widget_manager = \Drupal::getContainer()->get('plugin.manager.field.widget');
+    $definitions = $widget_manager->getDefinitions();
+
+    $widgets = [];
+    foreach ($definitions as $id => $definition) {
+
+      if (!in_array('entity_reference', $definition['field_types'])) {
+        continue;
+      }
+
+      $widgets[] = $id;
+    }
+
+    return $widgets;
+  }
```

Comment:
Yes - we would like to get all the available widget so the user could pick them through the UI. No?